core/services/keystore/csa.go

```diff
@@ -9,7 +9,8 @@ import (
 
 //go:generate mockery --name CSA --output mocks/ --case=underscore
 
-var ErrCSAKeyExists = errors.New("CSA key does not exist")
```

Comment:
This specific error value is being used in other files.
You might introduce a breaking change by modifying it.
I would suggest not to touch it.

Reply:
It is only being used as a check, and nobody was returning this error prior this changes. I just introduced the `return` of this error.